test/parallel/test-structuredClone-global.js

```diff
@@ -16,6 +16,27 @@ assert.strictEqual(structuredClone(undefined, null), undefined);
 assert.strictEqual(structuredClone(undefined, { transfer: null }), undefined);
 assert.strictEqual(structuredClone(undefined, { }), undefined);
 
+// ReadableStream subclass instance will be received as its closest transferable superclass
+{
+  const original = new ReadableStream();
+  const transfer = structuredClone(original, { transfer: [original] });
+  assert.strictEqual(Object.getPrototypeOf(transfer), ReadableStream.prototype);
```

Comment:
```suggestion
  assert.strictEqual(Object.getPrototypeOf(transfer), ReadableStream.prototype);
  assert.ok(transfer instanceof ReadableStrram);
```

Ditto for the rest

Reply:
sounds good. I noticed this was not exactly what the WPT was doing, so indeed updating it would be clearer.